src/main/java/io/temporal/samples/hello/HelloPeriodic.java

```diff
@@ -88,14 +96,33 @@
   // Define the workflow implementation which implements the greetPeriodically workflow method.
   public static class GreetingWorkflowImpl implements GreetingWorkflow {
 
+    // In this example, we use an internal of 10 seconds with an intended variation of +/- 2 seconds
+    // between executions of some useful work. In real applications a higher value may be more
+    // appropriate, for example one that matches a business cycle of several hours or even days.
+    private static class ScheduleConfig {
+      static final int PeriodTargetSecs = 10;
+      static final int PeriodVariationSecs = 4;
+    }
+
+    // The max history length of a single Temporal workflow is limited.
```

Comment:
I really like that we add comments, but we should be clear if possible. The limitation is 50K event upon which workflow is terminated. Saying "is limited" here imo is very confusing to new users as what does it mean, 10 seconds max? Just my 2c :)

Reply:
Addressed. Please take a look.